src/Security/Certificate.cs

```diff
@@ -263,19 +270,47 @@ public NSData GetPublicKey ()
 			}
 		}
 #else
-		[iOS (10,3)]
-		[TV (10,3)]
-		[Deprecated (PlatformName.iOS, 12,0)]
-		[Deprecated (PlatformName.TvOS, 12,0)]
-		[Deprecated (PlatformName.WatchOS, 5,0)]
+#if !NET
+		[iOS (10,3), TV (10,3)]
```

Comment:
Try to keep the original code.

In this case even looking at the PR without the whitespace changes is not helpful.